pywbem/cim_types.py

```diff
@@ -872,33 +875,33 @@ def type_from_name(type_name):
     """
     Return the Python type object for a given CIM data type name.
 
-    For example, type name "uint8" will return type object
+    For example, type name ``"uint8"`` will return type object
     :class:`~pywbem.Uint8`.
 
     For CIM data type names ``"string"`` and ``"char16"``, the
     :term:`unicode string` type is returned (Unicode strings are the preferred
     representation for these CIM data types).
 
-    The returned type can be used to 'cast' a value to that type in most cases,
-    except for "string" and "char16" typed values:
+    The returned type can be used as a constructor from a differently typed
+    input value, in many cases. Notable limitations are:
```

Comment:
Remove the comma  value, in. M change many to some.

Reply:
Comma: DONE.
"in many cases": They are really many cases for which the statement is true: All CIM integer types from an integer; all CIM real types from integer and float; CIMDateTime from Python datetime and timedelta and string in datetime format; CIM string and char16 from byte string except for the limitation, CIM reference from string in WBEM URI format and from CIMInstanceName.